README.md

```diff
@@ -2,21 +2,54 @@
 
 This demo shows how to build and run an AI meeting assistant based on Daily's transcription API alongside an embedded Prebuilt call.
 
-## Technical concept
+[ADD SCREENSHOT HERE]
```

Comment:
This README can be fleshed out in a followup once we have final UI screenshots etc